### PR TITLE
Globalize KeyValue Memory connector

### PR DESCRIPTION
**Pull request by superkhau**
[back-port of #1056]

@bajtos @0candy PTAL

### CODE REVIEW

File: intl/en/messages.json [hunk 1]
```diff
@@ -13,6 +15,7 @@
   "5ec8efeb715a2c34b440f2d76e2cf87d": "  {0}",
   "62a2d80c405b7fec5f547c448ab1b6ff": "The {{order}} {0} has invalid direction",
   "63f503acabdc4c7588aab741b8466862": "The {0} property has invalid clause {1}",
+  "67c2bf43b5281ab929617423ea8a6f3e": "The connector {0} does not support {{replaceById}} operation. This is not a bug in LoopBack. Please contact the authors of the connector, preferably via GitHub issues.",
```
Comment (superkhau) [inline intl/en/messages.json]: Unrelated, someone forgot to run `slt-globalize -e` for previous globalization PR.

File: intl/en/messages.json [hunk 2]
```diff
@@ -27,11 +30,11 @@
   "0be2d39d225b1d8b2a0f92ad5c65c9ac": "No model specified for {{polymorphic}} {0}: {1}",
   "2f4af31c144bbfab1bbf479866acd820": "\nWARNING: {{LoopBack}} connector \"{0}\" is not installed as any of the following modules:\n\n {1}\n\nTo fix, run:\n\n    {{npm install {2} --save}}\n",
   "6111399276924ffa3bc9a410cdfcb2e5": "No {{id}} name {0}",
+  "75518263527c423b8a12f908a2730b0f": "{{className}} is required",
```
Comment (superkhau) [inline intl/en/messages.json]: ditto

File: intl/en/messages.json [hunk 3]
```diff
@@ -27,11 +30,11 @@
...
   "791ab3031a73ede03f7d6299a85e8289": "Timeout in connecting after {0} ms",
   "7b277018e43d41bc445731092b91547d": "Not connected",
   "a2487abefef4259c2131d96cdb8543b1": "Connection fails: {0}\nIt will be retried for the next request.",
   "b15b20280211ad258d92947f05b6e4a5": "The connector has not been initialized.",
-  "ddf0aa14803f1c84f4a97f3803f7471c": "Class name required",
```
Comment (superkhau) [inline intl/en/messages.json]: ditto

File: intl/en/messages.json [hunk 4]
```diff
@@ -43,6 +46,7 @@
   "514985b2327f061ffb1c932f6b909979": "Model {0} is not defined.",
   "cc09e361de7b0f3dc1687515c5713885": "Property names containing a dot are not supported. Model: {0}, property: {1}",
   "da02dd6c53d4148320eeb31718a7aebe": "Invalid type for property {0}",
+  "ddf0aa14803f1c84f4a97f3803f7471c": "Class name required",
```
Comment (superkhau) [inline intl/en/messages.json]: ditto

Comment (bajtos) [inline intl/en/messages.json]: AFAICT, this line was moved from L34 to L49. This looks like a short-coming of strong-globalize, see https://github.com/strongloop/strong-globalize/issues/74

**Comment by 0candy** [comment]
LGTM

**Pull request merged**